source/package.json

```diff
@@ -1,18 +1,27 @@
 {
+  "name": "source",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest",
+    "lint": "eslint --fix public/**/*.js",
+    "doc": "jsdoc public -r -d docs"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
   "devDependencies": {
     "escape-string-regexp": "^5.0.0",
     "eslint": "^9.2.0",
     "jest": "^29.7.0",
+    "jest-puppeteer": "^6.1.0",
+    "puppeteer": "^19.0.0",
```

Comment:
Looks like this isn't the most recent version of Puppeteer, could you update it?